browserstack.json

```diff
@@ -4,15 +4,31 @@
 	"test_framework": "qunit",
 	"test_path": [
 		"test/index.html",
+		"test/startError.html",
 		"test/logs.html",
-		"test/amd.html"
+		"test/amd.html",
+		"test/autostart.html"
 	],
 	"browsers": [
 		"chrome_previous",
 		"chrome_current",
 		"firefox_previous",
 		"firefox_current",
+		"opera_previous",
 		"opera_current",
+		"safari_5_1",
+		{
+			"browser": "safari",
+			"browser_version": "6.0"
+		},
+		{
+			"browser": "safari",
+			"browser_version": "7.0"
+		},
+		{
+			"browser": "safari",
+			"browser_version": "7.1"
+		},
 		"safari_previous",
 		"safari_current",
```

Comment:
@mzgol 
This list run tests on Safari:

- 9 (current)
- 8 (previous)
- 7.1
- 7.0 (same Webkit as 6.2)
- 6.0
- 5.1

review, plz